examples/newrelicapm/otel-collector-config.yaml

```diff
@@ -22,16 +22,20 @@ exporters:
 
 connectors:
   newrelicapm:
+  newrelicapm/metrics:
 
 service:
   pipelines:
     traces:
       receivers: [otlp]
       processors: [newrelicapm, batch]
-      exporters: [newrelicapm, otlp, logging]
+      exporters: [newrelicapm, otlp]
     metrics:
       receivers: [otlp, newrelicapm]
       processors: [batch]
+      exporters: [otlp, newrelicapm/metrics]
+    metrics/apm:
```

Comment:
is it useful to have a different pipeline here? If not, I would vote for using a processor.

Reply:
I did not discover a way to do this without an additional pipeline. With a single pipeline the connector fails to start up because it detects a cycle. Keeping the config simple is the main reason I agree that a processor would be more ideal here. Ok to do in a follow up PR?